### PR TITLE
gnuski 0.3 (new formula)

### PR DESCRIPTION
**Pull request by tomyun**
[GNUSki](http://sourceforge.net/projects/gnuski/) is an open source clone of [SkiFree](https://en.wikipedia.org/wiki/SkiFree) that runs on the terminal.

### CODE REVIEW

File: gnuski.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  url "https://downloads.sourceforge.net/project/gnuski/gnuski/gnuski-0.3/gnuski-0.3.tar.gz"
+  sha256 "1b629bd29dd6ad362b56055ccdb4c7ad462ff39d7a0deb915753c2096f5f959d"
+
+  def install
+    system "make"
+    bin.install "gnuski"
+  end
+
+  test do
+    File.executable? "#{bin}/gnuski"
```
Comment (bfontaine) [inline gnuski.rb]: Please write a test for the actual functionality.

File: gnuski.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Gnuski < Formula
+  desc "Open source clone of Skifree"
+  homepage "http://gnuski.sourceforge.net/"
+  url "https://downloads.sourceforge.net/project/gnuski/gnuski/gnuski-0.3/gnuski-0.3.tar.gz"
+  sha256 "1b629bd29dd6ad362b56055ccdb4c7ad462ff39d7a0deb915753c2096f5f959d"
+
+  def install
+    system "make"
+    bin.install "gnuski"
```
Comment (bfontaine) [inline gnuski.rb]: Is there a `make install` we can use?

Comment (tomyun) [inline gnuski.rb]: No, there is no `install` target.

File: gnuski.rb (hunk 1 above)
Comment (tomyun) [inline gnuski.rb]: It's a very simple program that accepts no arguments. I'll remove `test` block then.

File: gnuski.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  end
```
Comment (bfontaine) [inline gnuski.rb]: Please add a `test do` block.

Comment (tomyun) [inline gnuski.rb]: I have no idea for writing a `test do` for programs that accept no arguments. Any suggestion?

**Comment by mistydemeo** [comment]
If the program is interactive-only, I think it's fine to have no test. Unfortunately common in this tap!

**Pull request closed without merging**